falcon/errors.py

```diff
@@ -379,9 +379,9 @@ class HTTPNotAcceptable(HTTPError):
             base articles related to this error (default ``None``).
     """
 
-    def __init__(self, description=None, headers=None, **kwargs):
+    def __init__(self, title='406 Not Acceptable', description=None, headers=None, **kwargs):
```

Comment:
I think we can just default *title* to ``None`` since HTTPError will default to using the status line as the title.